components/layout/AuthLayout.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useContext } from "react";
+import Head from "next/head";
+import { tw } from "../../utils";
+import { UIContext } from "../../context/ui";
+import { DarkModeIcon, LightModeIcon } from "../icons/Icons";
+
+interface HomeLayoutProps {
+  title: string;
+  pageDescription: string;
+  children: React.ReactNode;
+}
+
+export const AuthLayout: FC<HomeLayoutProps> = ({
+  title,
+  pageDescription,
+  children,
+}) => {
+  const { darkMode, activateDarkMode, deactivateDarkMode } =
+    useContext(UIContext);
+
+  return (
+    <div className={tw("h-screen ", darkMode ? "dark" : "")}>
```

Comment:
this works and there's nothing wrong with it, however I'd like to point out you can also do the same thing like this:

```
tw("h-screen", { dark : darkMode })
```